tests/test_all_rules.py

```diff
@@ -964,18 +964,14 @@ def test_event_override(self):
             # QueryRuleData should inheritenly ignore machine learning rules
             if isinstance(rule.contents.data, QueryRuleData):
                 rule_language = rule.contents.data.language
-                rule_integrations = rule.contents.metadata.get('integration')
-                if isinstance(rule_integrations, str):
-                    rule_integrations = [rule_integrations]
-                rule_query = rule.contents.data.get('query')
                 has_event_ingested = rule.contents.data.get('timestamp_override') == 'event.ingested'
                 rule_str = self.rule_str(rule, trailer=None)
 
                 if not has_event_ingested:
                     # TODO: determine if we expand this to ES|QL
                     # ignores any rule that does not use EQL or KQL queries specifically
                     # this does not avoid rule types where variants of KQL are used (e.g. new terms)
-                    if rule_language not in ('eql', 'kuery') or "sequence" in rule_query:
+                    if rule_language not in ('eql', 'kuery') or rule.contents.data.is_sequence:
```

Comment:
Now that I'm seeing this, when do we actually run into this case where we have an a language not eql but still is a sequence?

Reply:
Never, sequences are only in EQL.

Reply:
so do we even need the is_sequence check at all?